sos/report/plugins/ceph_iscsi.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class CephISCSI(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = "CEPH iSCSI"
+    plugin_name = "ceph_iscsi"
+    profiles = ("storage", "virt")
+
+    packages = ("ceph-iscsi",)
+    services = ("rbd-target-api", "rbd-target-gw")
+
+    def setup(self):
+        self.add_copy_spec(
+            [
+                "/etc/tcmu/tcmu.conf",
+                "/var/log/ceph/ceph-client.iscsi.log",
+                "/var/log/rbd-target-api/rbd-target-api.log",
+                "/var/log/rbd-target-gw/rbd-target-gw.log",
+                "/var/log/tcmu-runner.log",
+            ]
+        )
+
+        self.add_cmd_output(["gwcli info", "gwcli ls"])
+
+        self.add_forbidden_path(
+            [
+                "/etc/ceph/*keyring*",
+                "/var/lib/ceph/*keyring*",
+                "/var/lib/ceph/*/*keyring*",
+                "/var/lib/ceph/*/*/*keyring*",
+                "/var/lib/ceph/osd",
+                "/var/lib/ceph/mon",
+                # Excludes temporary ceph-osd mount location like
+                # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
+                "/var/lib/ceph/tmp/*mnt*",
+                "/etc/ceph/*bindpass*",
+            ]
```

Comment:
This is redundant here, as no such file is collected by this plugin.

Reply:
I'd happy to remove the section to be simpler. I assumed that it's an extra harness across all ceph plugins since radosgw plugin is in the same situation.
https://github.com/sosreport/sos/blob/089839231aa54261f274f9e963f2d07d3dc7de86/sos/report/plugins/ceph_rgw.py#L28-L39

Reply:
I see. Until @TurboTurtle opposes, let leave the section as is, then.

Reply:
We can afford to drop it in other plugins if it's superfluous - since none of the paths added by this particular plugin are in this forbidden list, we can drop this call as well.

Reply:
Done, dropped.